docs/api/nn.rst

```diff
@@ -1,11 +1,15 @@
-Neural Network Methods
-======================
+Neural Network Module
+=====================
 
-The ``libmultilabel.nn`` package contains two types of APIs for data processing and model training:
+The neural network module ``libmultilabel.nn`` contains three methods.
+The first two methods (i.e., ``libmultilabel.nn.data_utils`` and ``libmultilabel.nn.nn_utils``) are utilities for processing data and training a neural network model.
+The third method (``libmultilabel.nn.networks``) is a collection of classes that defines the neural networks.
 
 * `libmultilabel.nn.data_utils <../api/nn.html#libmultilabel-nn-data-utils>`_
 * `libmultilabel.nn.nn_utils <../api/nn.html#module-libmultilabel.nn.nn_utils>`_
+* `libmultilabel.nn.networks <../api/nn_networks.html>`_
```

Comment:
something is wrong here. Now in the index page on the left, nn.networks isn't within NN modules. Please check.